jumpscale/packages/admin/services/diskcheck.py

```diff
@@ -0,0 +1,23 @@
+from jumpscale.loader import j
+from jumpscale.tools.servicemanager.servicemanager import BackgroundService
+from jumpscale.tools.notificationsqueue.queue import LEVEL
+
+
+class DiskCheckService(BackgroundService):
+    def __init__(self, name="diskcheck", interval=60 * 60 * 12, *args, **kwargs):
+        """
+            Check disk space every 12 hours
+        """
+        super().__init__(name, interval, *args, **kwargs)
+
+    def job(self):
+        disk_obj = j.sals.fs.shutil.disk_usage("/")
+        free_disk_space = disk_obj.free // (1024.0 ** 3)
+        if free_disk_space <= 10:
```

Comment:
I think it can be done later, but all these services must be configurable, 10gb might be accepted in some setups.

Reply:
I'll add it as a todo. thanks